public/js/pimcore/document/edit.js

```diff
@@ -323,11 +323,21 @@ pimcore.document.edit = Class.create({
         Object.values(this.getEditables()).forEach(editable => {
             try {
                 if (editable.getName() && !editable.getInherited()) {
-                    let name = editable.getName();
-                    values[name] = {
-                        data: editable.getValue(),
-                        type: editable.getType()
-                    };
+                    const name = editable.getName();
+                    const type = editable.getType();
+                    const value = editable.getValue();
+                    if (type === "wysiwyg" &&
+                        (value === "<p><br></p>" || value === "<p><br data-mce-bogus=\"1\"></p>")) {
```

Comment:
@robertSt7 wouldn't it be better to use something more generic, like `empty(trim(strip_tags(value))` ?

Reply:
@brusch Thanks for the suggestion. I will create a PR for that